### PR TITLE
Feature/style improvements

### PR DESCRIPTION
**Pull request by ovcOS**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,6 +61,8 @@
     "react": "16.8.5",
     "react-dom": "16.8.5",
     "react-helmet": "5.2.0",
+    "typeface-quicksand": "0.0.71",
+    "typeface-roboto-slab": "0.0.72",
     "typeface-slabo-27px": "0.0.72",
     "typeface-work-sans": "0.0.72"
```
Comment (morloy) [inline package.json]: Remove ’em old font, man!

**Comment by bbinoy** [comment]
image
The hover is a bit strange on the left side of the button.

**Comment by bbinoy** [comment]
image

Here it would be nicer to not break the words. There's enough space anyway.

Also I'd reduce the line-height a little.

**Comment by bbinoy** [comment]
To be honest, I'm not a fan of these boxes around the logos:
image

**Comment by bbinoy** [comment]
Todo: Translations
image

File: package.json (hunk 1 above)
Comment (ovcOS) [reply package.json]: My bad, meant to do it and forgot. Done.

**Comment by ovcOS** [comment]
> image
> 
> Here it would be nicer to not break the words. There's enough space anyway.
> 
> Also I'd reduce the line-height a little.

Word break is fixed, and line-height reduced a bit

**Comment by ovcOS** [comment]
> To be honest, I'm not a fan of these boxes around the logos:
> image

I got rid of the borders. See how it looks now on hover. I personally don't mind it, but if you guys dislike it I can remove the box-shadow

**Comment by ovcOS** [comment]
I agree on the social icons, partially on the underline, but do you really want to change to this hover effect on the buttons?

**Pull request merged**